src/cmd/singularity/env_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/singularityware/singularity/src/pkg/test"
+)
+
+func TestSingularityEnv(t *testing.T) {
+	if os.Getenv("TEST_26") == "" {
```

Comment:
Should rather use the accepted way to skip disabled tests

```
	if !*runDisabled {
		t.Skip("disabled until issue addressed") // TODO
	}
```

Reply:
`go test --run_disabled ...` flag is defined in `src/cmd/singularity/cli_test.go`